superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { CHART_LIST } from './chart_list.helper';
+
+describe('chart card view', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.server();
+    cy.visit(CHART_LIST);
+  });
+
+  it('should load cards', () => {
+    cy.get('.chart-list-view');
+    cy.get('.ant-card').should('be.visible');
+    cy.get('.ant-card').should('have.length', 25);
+  });
+
+  it('should allow favor/unfavor chart card', () => {
```

Comment:
description nit: "favorite/unfavorite" - favor is either bias or a gift at a party.